client/src/components/timebutton/index.js

```diff
@@ -1,22 +0,0 @@
-import React, { Component } from "react";
```

Comment:
If you use `git mv` to move your files around instead of manually moving them in the file system, then your diff will just show the files as moved (and will show any changes that it finds too) instead of showing it as a whole file removed and another whole file added.
You'll also keep your git history of the file too, for example `blame` will know about changes that happened to the file before the move occurred.

Reply:
I never heard of `git mv`. It was annoying to make about 40 file changes just to rename some folders. this is very clean :smile: